README.md

```diff
@@ -1,2 +1,46 @@
 # Hello-World
```

Comment:
Please use [this](https://github.com/microverseinc/readme-template) template for README from Microverse. README should be professional.